src/java.base/share/classes/java/util/spi/LocaleServiceProvider.java

```diff
@@ -152,6 +152,9 @@
  * supported by the Java runtime environment. The following table lists the
  * version of CLDR used in each JDK release. Unless otherwise specified, all
  * update releases in a given JDK release family use the same CLDR version.
+ * Note that the CLDR locale data are subject to change. Users should not assume
+ * that the locale data remain the same across CLDR versions. Otherwise, unexpected
+ * incompatible behaviors may occur, such as an exception on parsing a date.
```

Comment:
For those wondering what has changed from one version to another or whether a particular change might have an impact after reading this note, I wonder if some kind of notes or links might be helpful. A column to the table below to add some notes? I'm sure they can always look at the CLDR release notes. Just wonder if there's anything specific worth noting or a link.

Reply:
right.. there's never "no change".. and any change could potentially break users if they are expecting identical behavior.